frontend/tests/unit/services/transactions/request-manager.spec.ts

```diff
@@ -320,7 +340,7 @@ describe('request-manager', () => {
 
       expect(
         getRequestIdentifier(RPC_URL, REQUEST_MANAGER_ADDRESS, transactionHash),
-      ).rejects.toThrow("Request Failed. Couldn't retrieve Request ID");
+      ).rejects.toThrow('Transaction reverted on chain.');
```

Comment:
This message is a bit missleading in this case then. It should throw "Request Failed. Couldn't retrieve Request ID" when there were no logs found in the receipt.logs right?